app/src/main/java/com/animal_villa/dto/Animal.kt

```diff
@@ -1,4 +1,7 @@
 package com.animal_villa.dto
 
-class Animal {
+data class Animal(var name : String, var reputation : Int) {
+    override fun toString(): String {
+        return "$name $reputation"
+    }
```

Comment:
Basic Animal data class. Allows you to create and instantiate the class, as well as a toString to return the name and reputation. Here is a handy article on data classes: https://kotlinlang.org/docs/data-classes.html